server_installation/topics/clustering/load-balancer.adoc

```diff
@@ -5,6 +5,11 @@ This section discusses a number of things you need to configure before you can p
 in front of your clustered {project_name} deployment.  It also covers configuring the built in load balancer that
 was <<_clustered-domain-example, Clustered Domain Example>>.
 
+The following diagram illustrates the use of a load balancer between clients and a {project_name} cluster. In another scenario, the load balancer could be a reverse proxy server.
```

Comment:
Perhaps a better formulation would be:

"The following diagram illustrates the use of a load balancer. The load balancer serves as a reverse proxy between the clients and the {project_name} servers."

Reply:
Thanks, @tkyjovsk.  I have made that change.